x-pack/plugins/security_solution/yarn.lock

```diff
@@ -1 +0,0 @@
-../../../yarn.lock
```

Comment:
Note: this was able to be removed since there are no longer any non-dev dependencies for it's package.json.